comp/core/config/params.go

```diff
@@ -94,9 +94,10 @@ func NewSecurityAgentParams(securityAgentConfigFilePaths []string, options ...fu
 }
 
 func NewClusterAgentParams(configFilePath string, options ...func(*Params)) Params {
-	return NewParams(common.DefaultConfPath,
-		WithConfFilePath(configFilePath),
-		WithConfigName("datadog-cluster"))
+	params := NewParams(common.DefaultConfPath, options...)
```

Comment:
Why not adding the two funcs at the end of `options`? In case the function does something more than setting the value?

Reply:
I basically just look at how it was done in similar functions of the same file in order to keep an homogeneous coding style.
Like https://github.com/DataDog/datadog-agent/blob/dc88d14851354cada1d15265220a39dce8840dcc/comp/core/config/params.go#L73-L78 for ex.